pandas/tests/io/json/test_readlines.py

```diff
@@ -252,3 +252,30 @@ def test_readjson_lines_chunks_fileurl(datapath):
     with pd.read_json(file_url, lines=True, chunksize=1) as url_reader:
         for index, chuck in enumerate(url_reader):
             tm.assert_frame_equal(chuck, df_list_expected[index])
+
+
+def test_chunksize_is_incremental():
+    jsonl = (
```

Comment:
can you add the issue number as a comment here

Reply:
Done.